### PR TITLE
487 activity cards refinements

### LINKED ISSUE #487: Activity Cards – Visual Refinements
Hi @Juliana-t @katevoss @megganeturner,

I've conducted a visual review of the latest release Activity Cards screens.

I have marked up some small visual refinements that I would recommend implementing from a FED perspective so that we align as closely as possible with the new design system.

Please see comments pinpointing issues on the Invision Prototype:

• https://invis.io/W810BV0PMFSK
• https://invis.io/ZY10BV0TQTKE
• https://invis.io/PJ10BV0U3SKD
• https://invis.io/3810BV0VAVCP

Hope this helps and drop me a line with any queries at all.

Thanks again, Andy

---

### InVision summary

- [ ] [Reduce space between page number and progress bar](https://projects.invisionapp.com/share/W810BV0PMFSK#/screens/445952099/comments/140924391)
- [ ] [Fully rounded progress bar](https://projects.invisionapp.com/share/W810BV0PMFSK#/screens/445952099/comments/140924408)
- [ ] [Make bottom buttons always visible](https://projects.invisionapp.com/share/W810BV0PMFSK#/screens/445952099/comments/140924378)
- [ ] [Align "X" button to right](https://projects.invisionapp.com/share/ZY10BV0TQTKE#/screens/445952100/comments/140924476)
- [ ] [Reduce space between title and sub-title](https://projects.invisionapp.com/share/ZY10BV0TQTKE#/screens/445952100/comments/140924442)
- [ ] [Review button padding](https://projects.invisionapp.com/share/ZY10BV0TQTKE#/screens/445952100/comments/140924466)
- [ ] [Padding between subtitle and selectors](https://projects.invisionapp.com/share/PJ10BV0U3SKD#/screens/445952101/comments/140924514)
- [ ] [Space between buttons](https://projects.invisionapp.com/share/PJ10BV0U3SKD#/screens/445952101/comments/140924530)
- [ ] [Button size inconsistency](https://projects.invisionapp.com/share/PJ10BV0U3SKD#/screens/445952101/comments/140924556)
- [ ] [Increase padding between title and copy](https://projects.invisionapp.com/share/3810BV0VAVCP#/screens/445952102/comments/140924592)
- [ ] [Increase padding between copy and button](https://projects.invisionapp.com/share/3810BV0VAVCP#/screens/445952102/comments/140924603)

### PR DESCRIPTION
**Pull request by megganeturner**
Closes #487

- Reduce space between page number & progress bar + re-centre the resources heading in topmenu
- Tweaks to spacing around buttons & headings
- Make the cards scrollable - so you no longer have to scroll down for the 'next' button on a lesson card with long content! 🎉 

giphy (3)

### CODE REVIEW

**Review by PeteCoward — COMMENTED** [review]
looks good to me ( I may publish it for review/demo ) but will leave it to Kate to review

File: src/riot/Lesson/LessonComplete.riot.html [hunk 1]
```diff
@@ -21,9 +21,7 @@ <h2>{ TRANSLATIONS.good() }</h2>
                 { TRANSLATIONS.exam() }
                 <span class="chevron"></span>
             </button>
-            <button class="btn-secondary" onclick="{goToCourseOverview}">
-                { TRANSLATIONS.course() }
-            </button>
+            <a href="#{state.lesson.course.loc_hash}">{ TRANSLATIONS.course() }</a>
```
Comment (katevoss) [inline src/riot/Lesson/LessonComplete.riot.html:24]: This button isn't working:
image
I tried a few different lessons, the "Next Lesson" button doesn't work.

**Review by katevoss — COMMENTED** [review]
Looks real slick!

One other issue, in LessonPage:
image
Gotta switch to using `getHtmlUrl` from `utilities.js` I think.

File: src/riot/Lesson/LessonComplete.riot.html (hunk 1 above)
Comment (megganeturner) [reply src/riot/Lesson/LessonComplete.riot.html:24]: I have no idea why I deleted the code for this button, but it's back now 🙃

**Comment by PeteCoward** [comment]
merging to start the week with latest deployed

**Pull request merged**